tests/test_application.py

```diff
@@ -76,3 +76,14 @@ def test_jump_handler(app, test_client, referer, jump_link, expect):
             expect)
         assert result.status_code == 302
         assert get_url == expect_url
+
+
+def test_incognito(app, test_client):
+    with app.test_request_context():
+        app.preprocess_request()
+        flask.g.ledger.fava_options['incognito'] = True
```

Comment:
This should probably be reset after this test as the ledger object will be shared between tests